Makefile

```diff
@@ -96,7 +96,7 @@ help:                       ## Display this help message.
 	awk -F ':.*?## ' 'NF==2 {printf "  %-26s%s\n", $$1, $$2}'
 
 test: env                   ## Run all tests.
```

Comment:
let's create separate item for race test.

Reply:
Done.